charts/thoras/templates/api-server/deployment.yaml

```diff
@@ -42,6 +42,10 @@ spec:
               secretKeyRef:
                 name: thoras-elastic-password
                 key: host
+          - name: SLACK_WEBHOOK_URL
+            value: {{ .Values.thorasApiServer.slackWebhookUrl | default .Values.slackWebhookUrl }}
```

Comment:
This is my first time editing helm templates. [The documentation](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) seemed to favor piping things through values, but I notice that we don't do that for `default` in other places in the templates. I'm open to feedback here.

Reply:
Nope, I imagine the way we've been doing it historically is legacy syntax. Thanks for leveling up our best practices!